src/pages/HomePage/AuthForms/Forms.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import HowToRegIcon from '@material-ui/icons/HowToReg';
+import AddToHomeScreenIcon from '@material-ui/icons/AddToHomeScreen';
+import SignIn from "./Login";
+import Register from "./Register";
+import styled from 'styled-components'
+
+const Forms: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const [LoginForm, SetLoginForm] = React.useState(false);
+
+  const trigger = (value: boolean) => {
+    SetLoginForm(value);
+  };
+
+  const handleClickOpen = () => {
```

Comment:
Do we need that function? I think we can have one function responsible for opening the modal and it will handle changing both states.

```
const handleOpen = (isLogin: boolean) => {
  SetLoginForm(isLogin);
  setOpen(true);
}
```